cmd/lakectl/cmd/abuse.go

```diff
@@ -50,7 +50,7 @@ var abuseRandomReadsCmd = &cobra.Command{
 	Hidden: false,
 	Args:   cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		u := MustParseRefURI("source ref", args[0])
+		u := MustParseRef("source ref", args[0])
```

Comment:
You're right, I removed the URI part as I thought it might be confusing with the uri.Parser package being used internally.
The function initially was testing if the URI is valid, regardless if LakeFS can really use it; I included the RegEx validation that was located in the API package to the same flow to make the validation more strict.

Probably it would make more sense to leave it as it, reverting this one, thanks.